assets/src/blocks/Counter/CounterBlock.js

```diff
@@ -52,45 +52,46 @@ export class CounterBlock {
           save() {
             return null;
           },
-        }
+        },
       ],
-      edit: ( { isSelected, attributes, setAttributes } ) => {
+      // eslint-disable-next-line no-shadow
+      edit: ({isSelected, attributes, setAttributes}) => {
         return <CounterEditor
           attributes={attributes}
           setAttributes={setAttributes}
-          isSelected={ isSelected }
-        />
+          isSelected={isSelected}
+        />;
       },
-      save: frontendRendered( BLOCK_NAME )
+      save: frontendRendered(BLOCK_NAME),
     });
 
     // Remove the default style since it's the same as "text only"
     unregisterBlockStyle(BLOCK_NAME, 'default');
 
     const styles = [
-        {
-          name: 'plain',
-          label: 'Text Only',
-          isDefault: true
-        },
-        {
-          name: 'bar',
-          label: 'Progress Bar'
-        },
-        {
-          name: 'arc',
-          label: 'Progress Dial'
-        },
+      {
+        name: 'plain',
+        label: 'Text Only',
+        isDefault: true,
+      },
+      {
+        name: 'bar',
+        label: 'Progress Bar',
+      },
+      {
+        name: 'arc',
+        label: 'Progress Dial',
+      },
 
-      ]
+    ];
 
     if (window.p4ge_vars.features.feature_engaging_networks) {
       styles.push({
         name: 'en-forms-bar',
-        label: __('Progress Bar inside EN Form', 'planet4-blocks-backend')
+        label: __('Progress Bar inside EN Form', 'planet4-blocks-backend'),
```

Comment:
I'm pretty sure that this should not be translatable, like the other styles 🤔

Reply:
Should I remove it, because I have no idea too.

Reply:
Yes I think you can remove the translation function there